package.json

```diff
@@ -195,6 +215,21 @@
         "command": "one.viewer.jsonTracer",
         "title": "json tracer",
         "category": "ONE"
+      },
+      {
+        "command": "one.viewer.metadata.show",
+        "title": "Show Metadata",
+        "category": "ONE"
+      },
+      {
+        "command": "one.viewer.relation.show",
+        "title": "Show Relation",
+        "category": "ONE"
+      },
+      {
+        "command": "one.metadata.updateViewerState",
```

Comment:
Could you remove these `Metadata Viewer`-related lines? (3)

Reply:
ok, I`ll apply that part.